release-packaging/Classes/FluidBufToKr.sc

```diff
@@ -0,0 +1,19 @@
+FluidKrToBuf {
+	*kr {
+		arg krStream, buffer;
+		^buffer.numFrames.do{
+			arg i;
+			BufWr.kr(krStream[i], buffer, i);
+		}
+	}
+}
+
+FluidBufToKr {
+	*kr {
+		arg buffer;
+		^buffer.numFrames.collect{
```

Comment:
Do we need to be wary of what happens if this gets called on a very big buffer? Presumably the number of possible outputs is capped at some point?

Reply:
Since it will try to make a really big krStream, it will probably need to write the SynthDef to file before sending to the server, but then it should probably run fine. Do you want me to test this with a huge buffer?

Reply:
I guess it either (a) works (b) fails to build the SynthDef or (c) seems to work but crashes the server!

In the case of (b) and (c) it hinges on how hard it is to diagnose what went wrong. So, yes, probably worth checking and if it is error-prone (esp. if it can crash), we should see about making it easy to diagnose or avoid

Reply:
Ah yes, if someone tries to pass in a buffer that is like the length of an audio file, that will definitely be a problem. I put in a check that if the buffer is > 1000 samples long, it will throw an error.